dnsimple/dnsimple-developer
# Move to dart sass

This PR removes Sass in favor of Dart Sass.

Closes https://github.com/dnsimple/dnsimple-developer/issues/175

## QA

1. `rake run`
2. Open site
    - Layout, colors and font should match developer.dnsimple.com
    - Search a topic: matches developer.dnsimple.com
    - Browse an article: matches developer.dnsimple.com
